content/api-reference/queries/search-activities.mdx

```diff
@@ -10,8 +10,12 @@ description: Search activities by status, type, patient and other criteria
 ## Query
 
 ```graphql
-query activities($filters: FilterActivitiesParamsGraphqlType) {
-  activities(filters: $filters) {
+query activities(
+  $filters: FilterActivitiesParamsGraphqlType
+  $pagination: PaginationParamsGraphqlType
+  $sorting: SortingParamsGraphqlType
```

Comment:
```suggestion
  $filters: FilterActivitiesParams
  $pagination: PaginationParams
  $sorting: SortingParams
```

Reply:
In the backend, we call them *GraphqlType to disambiguate with the interfaces they implement, but they are exposed without the suffix. I believe this is what you should be seeing via introspection as well